### ISSUE TITLE
Unusabled with typescript

### ISSUE THREAD

**Issue by RyRy79261** [wangzuo/react-input-number#16]
## Description

After manually declaring the `.d.ts` file to compile, the component renders a basic, `type="text"` input with no additional functionality

**Comment by wangzuo** [comment]
pr welcome to add the typescript definition file

**Comment by RyRy79261** [comment]
There is nothing to add as the package outputs the following markup:
`<input type=text />` hence the title

**Comment by wangzuo** [comment]
dont know what you mean, but you can always add type definition for the a react component to support it in typescript

**Comment by RyRy79261** [comment]
@wangzuo This component is meant to create a number input, however, the only functionality being output buy this component, is a minimalistic text input, this component doesn't function or do anything or add any additional features when using via typescript. 

**This:**: `<InputNumber min={10} max={100} step={0.03} value={num} onChange={setNum} />`
**Outputs to this:** `<input type=text />`

**Comment by RyRy79261** [comment]
Tl;Dr: it doesn't work
